types/lodash/common/object.d.ts

```diff
@@ -1951,15 +1951,15 @@ declare module "../index" {
          * _.omitBy(object, _.isNumber);
          * // => { 'b': '2' }
          */
-        omitBy<T>(object: Dictionary<T> | null | undefined, predicate?: ValueKeyIteratee<T>): Dictionary<T>;
+        omitBy<T extends object>(object: T | null | undefined, predicate: ValueKeyIteratee<T[keyof T]>): {[K in keyof T]?: T[K]};
```

Comment:
Is the return type somehow different from `PartialObject<T>` (a.k.a. `Partial<T>`)? If not, use `PartialObject<T>` so it's more readable.